### PR TITLE
workers-4 Legislation fetch and save

### PR DESCRIPTION
**Pull request by greenac**

### CODE REVIEW

File: fetchControllers/bill.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+			count += 1
+		} else {
+			logger.Warn("Failed to save bill:", bill.Title, "to the database")
+		}
+	}
+
+	return count, nil
+}
+
+func (bfc *BillFetchController) FetchAndSaveRecentSenateBills(congress int) (int, error) {
```
Comment (aravind-lattis) [inline fetchControllers/bill.go]: @greenac we are fetching only passed bills is that okay? They just return 20 bills which has passed recently?

File: fetchControllers/bill.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	return bfc.fetchAndSaveAllBills(congress, fetchers.Senate, fetchers.Passed)
```
Comment (thearavind) [inline fetchControllers/bill.go:68]: @greenac We are fetching only passed bills is that okay? They just return 20 bills which has passed recently

**Comment by thearavind** [comment]
@greenac Should we use camel case or underscore's for file names and package names? now `fetchers/member_fetcher.go` , `fetchControllers/member.go` both exists

File: fetchControllers/bill.go (hunk 2 above)
Comment (greenac) [reply fetchControllers/bill.go:68]: @thearavind I think it's ok for now. We'll probably want to get more sophisticated in the future. I think to get us started though, this is fine.

Comment (greenac) [reply fetchControllers/bill.go:68]: @thearavind I think it's ok for now. We'll probably want to get more sophisticated in the future. I think to get us started though, this is fine.

**Pull request merged**